recipes/libphonenumber/all/conanfile.py

```diff
@@ -0,0 +1,195 @@
+import os
+
+from conan import ConanFile, conan_version
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.apple import is_apple_os
+from conan.tools.build import check_min_cppstd, can_run
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.env import VirtualBuildEnv
+from conan.tools.files import copy, get, rm, rmdir, replace_in_file
+from conan.tools.gnu import PkgConfigDeps
+
+required_conan_version = ">=1.56.0 <2 || >=2.0.6"
+
+
+class LibphonenumberConan(ConanFile):
+    name = "libphonenumber"
+    description = "Google's common C++ library for parsing, formatting, and validating international phone numbers"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/google/libphonenumber"
+    topics = ("phone-numbers",)
```

Comment:
Maybe `phone` should be a separate topic, too?

```suggestion
    topics = ("phone", "phone-numbers",)
```

Reply:
Added.

Reply:
Thanks!